package.json

```diff
@@ -138,7 +137,7 @@
     "server": "node scripts/server.js",
     "server:development": "nodemon $NODE_DEBUG_OPTION scripts/server.js --watch build/render-server.js",
     "bdd-test": "node ./node_modules/.bin/wdio wdio.conf.js",
-    "eslint": "./node_modules/.bin/eslint .",
+    "eslint": "./node_modules/.bin/eslint apps",
```

Comment:
wouldn't it be better to use the eslintignore?

folders like fixtures is ignored now